doc/version-history.rst

```diff
@@ -2,6 +2,12 @@
 Version History
 ===============
 
+=======
+v0.5.2
```

Comment:
since you are adding an entire submodule, we should probably have a minor release instead of a patch... Patch releases are for small backward compatible changes.